### ISSUE TITLE
Web vitals usage

### ISSUE THREAD

**Issue by grzanka** [yaptide/ui#1637]
We recently bumped web-vitals (https://www.npmjs.com/package/web-vitals) to 4.0.0: https://github.com/yaptide/ui/pull/1629

I am however not sure how the metrics reported by web-vitals are used. Is there any usage of it ?

**Comment by grzanka** [comment]
Does it work with vite ?

**Comment by NiEkOKsU** [comment]
Yes, web-vitals works with vite, we don't need any changes in code to achieved same result as in CRA application.

**Comment by grzanka** [comment]
Do we have any piece of documentation how to use web-vitals ?

**Comment by NiEkOKsU** [comment]
At the moment we have only comments in src/index.tsx file. It is a good idea to add it to a documentation, if such would be exists :)
